rust/hyperlane-base/src/settings/chains.rs

```diff
@@ -336,12 +336,7 @@ impl ChainSetup {
                 functions: EthereumInterchainGasPaymasterAbi::fn_map_owned(),
             });
         }
-        if let Ok(addr) = self.addresses.multisig_ism.parse() {
-            cfg.contracts.entry(addr).or_insert_with(|| ContractInfo {
-                name: Some("msm".into()),
-                functions: EthereumMultisigIsmAbi::fn_map_owned(),
-            });
-        }
+        // TODO: How/where do I add this entry?
```

Comment:
Hmm now I'm thinking about how `PrometheusMiddleware` will keep track of metrics for all eth_calls that we make. The contract address is included in this metric as a label: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/ethers-prometheus/src/middleware/mod.rs#L409

Now that we're calling the `formatMetadata` function on the recipient's ISM, we could find ourselves with a lot of unique addresses and a lot of metric bloat. The easiest options to me are probably:
1. Don't make any changes, but just be aware that we may want to change this in the future if we start seeing people actually use their own ISMs
2. Don't specify the actual contract address (i.e. set the zero address or something) in situations where we're calling a contract that isn't specified in the `PrometheusMiddlewareConf`

I'm leaning toward 1, wdyt?

Reply:
Yeah I'd lean towards (1) as well for now, especially since we're still a bit away from non-default ISMs being practical.

In time it could be interesting to see how many ISMs are out there being used.